core/primitives/babe_configuration.hpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_BABE_CONFIGURATION_HPP
+#define KAGOME_CORE_PRIMITIVES_BABE_CONFIGURATION_HPP
+
+#include "common/blob.hpp"
+#include "consensus/babe/common.hpp"
+#include "crypto/sr25519_types.hpp"
+#include "primitives/authority.hpp"
+
+namespace kagome::primitives {
+
+  using BabeSlotNumber = uint64_t;
+  using BabeClock = clock::SystemClock;
+  using BabeDuration = BabeClock::Duration;
+  using Randomness = common::Blob<crypto::constants::sr25519::vrf::OUTPUT_SIZE>;
+
+  /// Configuration data used by the BABE consensus engine.
+  struct BabeConfiguration {
+    /// The slot duration in milliseconds for BABE. Currently, only
+    /// the value provided by this type at genesis will be used.
+    ///
+    /// Dynamic slot duration may be supported in the future.
+    BabeDuration slot_duration;
+
+    BabeSlotNumber epoch_length;
+
+    /// A constant value that is used in the threshold calculation formula.
+    /// Expressed as a rational where the first member of the tuple is the
+    /// numerator and the second is the denominator. The rational should
+    /// represent a value between 0 and 1.
+    /// In the threshold formula calculation, `1 - c` represents the probability
+    /// of a slot being empty.
+    std::pair<uint64_t, uint64_t> c;
```

Comment:
I understand why this name was chosen and it is reasonable, but still I suggest to consider renaming it to something more meaningful without the comment above.

Reply:
any suggestions?

Reply:
I would prefer to keep the same naming as in substrate